samples/tekton/pac/tasks/gather-deploy-images.yaml

```diff
@@ -48,7 +48,8 @@ spec:
           image=$(yq "$IMAGE_PATH" "$yaml_path")
 
           if [ -n "$TARGET_BRANCH" ]; then
-            prev_image=$(git show "origin/$TARGET_BRANCH:$yaml_path" | yq "$IMAGE_PATH")
+            git fetch origin $TARGET_BRANCH
+            prev_image=$(git show "FETCH_HEAD:$yaml_path" | yq "$IMAGE_PATH")
```

Comment:
This is fixing shallow copy issue introduced in last pipeline synch. Also fixed here: https://github.com/redhat-appstudio/tssc-sample-pipelines/pull/105